### DISCUSSION TITLE
Setup MailGun or Mail Chimp

### DISCUSSION THREAD

**Discussion by chamodshehanka** [sliit-foss/rsvp#26] [Selection]

**Comment by sithumonline** [comment]
We can go with [nodemailer](https://nodemailer.com/about/) What's your opinion @Shehanka ?

**Comment by chamodshehanka** [comment]
is this done? @ebonynon

**Comment by sithumonline** [comment]
Yup, but we have to do implementation. It has a separate issue #6

**Reply by chamodshehanka** [reply]
Then do it.
